fetch.bs

```diff
@@ -5207,19 +5208,36 @@ these steps:
 
     <dl class=switch>
      <dt>"<code>error</code>"
-     <dd><p>Set <var>response</var> to a <a>network error</a>.
+     <dd>
+      <ol>
+       <li><p>Set <var>response</var> to a <a>network error</a>.
+      </ol>
```

Comment:
😂 I'll allow it, but please put it all on one line when there's not multiple children?